src/modules/75-cd/components/PipelineSteps/TerragruntPlan/InputSteps/TgPlanInputStep.tsx

```diff
@@ -102,6 +102,25 @@ function TgPlanInputStep(props: TerragruntPlanProps & { formik?: FormikContextTy
           />
         </div>
       )}
+
+      {getMultiTypeFromValue(get(inputSetData?.template, 'spec.configuration.moduleConfig.path')) ===
+        MultiTypeInputType.RUNTIME && (
+        <div className={cx(stepCss.formGroup, stepCss.md)}>
+          <TextFieldInputSetView
+            placeholder={'Enter path'}
+            label={getString('common.path')}
+            name={`${pathPrefix}spec.configuration.moduleConfig.path`}
+            disabled={readonly}
+            template={inputSetData?.template}
+            fieldPath={'spec.configuration.moduleConfig.path'}
+            multiTextInputProps={{
+              expressions,
+              allowableTypes
+            }}
+          />
+        </div>
```

Comment:
`TextFieldInputSetView` has an internal grid so wrapping div is not required. You can just pass the classNames.
```suggestion
        <TextFieldInputSetView
          placeholder={'Enter path'}
          label={getString('common.path')}
          name={`${pathPrefix}spec.configuration.moduleConfig.path`}
          disabled={readonly}
          template={inputSetData?.template}
          fieldPath={'spec.configuration.moduleConfig.path'}
          multiTextInputProps={{
            expressions,
            allowableTypes
          }}
          className={cx(stepCss.formGroup, stepCss.md)}
        />
```